packages/design/src/physics/characterkinematic/ICharacterControllerManager.ts

```diff
@@ -0,0 +1,56 @@
+import { Vector3 } from "@oasis-engine/math";
+import { ICharacterController } from "./ICharacterController";
+import { IColliderShape } from "../shape";
+
+/**
+ * Manages an array of character controllers.
+ */
+export interface ICharacterControllerManager {
```

Comment:
Delete this and add createController to IPhysXManager;

Reply:
done